### ISSUE TITLE
AnimationController.animateTo() called after AnimationController.dispose()

### ISSUE THREAD

**Issue by bingxiheart** [mjl0602/left-scroll-actions#14]
════════ Exception caught by foundation library ════════════════════════════════════════════════════
The following assertion was thrown while dispatching notifications for LeftScrollStatus:
AnimationController.animateTo() called after AnimationController.dispose()
AnimationController methods should not be used after calling dispose.
'package:flutter/src/animation/animation_controller.dart':
Failed assertion: line 513 pos 7: '_ticker != null'

When the exception was thrown, this was the stack: 
#2      AnimationController.animateTo (package:flutter/src/animation/animation_controller.dart:513:7)
#3      CupertinoLeftScrollState.close (package:left_scroll_actions/cupertinoLeftScroll.dart:268:27)
#4      CupertinoLeftScrollState.handleChange (package:left_scroll_actions/cupertinoLeftScroll.dart:110:7)
#5      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:226:25)
#6      ValueNotifier.value= (package:flutter/src/foundation/change_notifier.dart:292:5)
...
The LeftScrollStatus sending notification was: LeftScrollStatus#18e11(false)

**Comment by ThinkCats** [comment]
+1 ,same issue。只要切换到其他的页面，再回到 cupertinoLeftScroll 的页面中时，就可以触发。
应该是切换到其他页面后， AnimationController 已经被dispose了。
可以在以下代码的问题点，判断一下 animationController 的状态，我是一个新手，不知道是不是判断 animationController 是null就可以了。

问题所在文件的： left_scroll_actions-2.2.0/lib/cupertinoLeftScroll.dart：
```
// 关闭
  void close() {
    if (translateX != 0) {
      animationController.animateTo(0);  //here 问题点
    }
    if (widget.closeTag == null) return;
    if (_ct!.value == LeftScrollStatus.open) {
      _ct!.value = LeftScrollStatus.close;
    }
  }
```

**Comment by mjl0602** [comment]
你好，感谢您的关注，应该是判断下if(mouted)就好了，看报错不舒服的话可以先去源码里加上。我这边会在测试后更新
```dart
if(mounted) animationController.animateTo(0);
```

**Comment by mjl0602** [comment]
Fixed in 2.3.0 
修复AnimationController使用后未释放的错误，新增LeftScrollController API和LSTag类，更新示例和依赖版本
